src/Middleware/ConcurrencyLimiter/src/ConcurrencyLimiterMiddleware.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.AspNetCore.ConcurrencyLimiter;
 /// <summary>
 /// Limits the number of concurrent requests allowed in the application.
 /// </summary>
+[Obsolete("Concurrency Limiter middleware has been deprecated, use new rate limiting middleware. See: aspnet/rate-limiting")]
```

Comment:
```suggestion
[Obsolete("Concurrency Limiter middleware has been deprecated and will be removed in a future release. Update the app to use concurrency features in rate limiting middleware. For more information, see https://aka.ms/aspnet/rate-limiting")]
```

aka.ms link goes to `https://learn.microsoft.com/aspnet/core/performance/rate-limit` (note: no en-us in the link so user is redirected based on browser culture).

Use on all obsoletes.